packages/bundler-plugin-core/src/sentry/sourcemaps.ts

```diff
@@ -19,52 +61,46 @@ export function getFiles(includePath: string, includeEntry: InternalIncludeEntry
     return [];
   }
 
-  let files: {
-    absolutePath: string;
-    // Contains relative paths without leading `.` (e.g. "foo/bar.js" or "asdf\\a\\b.js.map")
-    relativePath: string;
-  }[];
+  let files: FileNameRecord[];
```

Comment:
Is there a reason we define a `let` here instead of just returning directly in the if-else?

Reply:
Nope, no particular reason. Changed it in [9253137](https://github.com/getsentry/sentry-javascript-bundler-plugins/pull/80/commits/9253137ab8ccde3a2602eee782f3054d3d01b212)